account_analytic_distribution_manual/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+#. Go to Invoicing > Analytic Accounting > Manual analytic distributions
```

Comment:
Go to Invoicing > Configuration > Analytic Accounting > Manual Analytic Distributions